__tests__/app.test.js

```diff
@@ -25,4 +25,31 @@ describe("GET /api/categories", () => {
             })
         })
     })
+})
+
+describe("GET /api/reviews", () => {
+    test("200: returns an array of objects, including comment_count and sorted by descending order", () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(( { body } ) => {
+            const reviews = body.reviews
+            expect(reviews).toBeInstanceOf(Array)
+            expect(reviews).toHaveLength(13)
```

Comment:
Like the idea of checking the array length here before you loop round the properties to prevent any false positives if the array was empty!